scripts/dockerfiles/run.sh

```diff
@@ -78,9 +78,17 @@ service rsyslog start
 cron
 
 # Run custom post config role scripts
+# First run all the scripts called pre-start
+for fname in $(find /var/vcap/jobs/*/bin -name pre-start) ; do
+    bash $fname
+done
+# And run any custom scripts other than pre-start
+
 {{ range $script := .role.PostConfigScripts}}
+{{ if not (is_pre_start $script) }}
```

Comment:
We should fix the indenting here so it's easier to read in the template. (It'll have extra indenting in the emitted shell script, but that's okay)